x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/edit_flyout/overrides.js

```diff
@@ -499,6 +522,29 @@ class OverridesUI extends Component {
             ))}
           </React.Fragment>
         )}
+
+        {format === FILE_FORMATS.SEMI_STRUCTURED_TEXT && originalGrokFieldNames.length > 0 && (
```

Comment:
Is this check working for Apache log files? Guess it should be hidden for this case?

![image](https://user-images.githubusercontent.com/7405507/148388450-72a0ee5a-c102-4a06-9edb-baa193ccd196.png)

Reply:
This is broken. Grok patterns containing a single pattern should not show a field list

Reply:
Fixed in 1c588c4516cf522eac1d0fd0d966d8155a37cb79